client/src/pages/Testing/RunTest.tsx

```diff
@@ -63,6 +63,7 @@ export function RunTest({
         <span>
           To download test files for some conditions you can
           <a
+            target="_blank"
```

Comment:
decided to add this one liner to [fix this](https://app.zenhub.com/workspaces/dibbs-ecr-refiner-67ddd053d70b9f000ffbb542/issues/gh/cdcgov/dibbs-ecr-refiner/521)